src/controllers/activegate/capability/capability.go

```diff
@@ -300,10 +306,11 @@ func statsdIngestBase() *capabilityBase {
 		shortName: dynatracev1beta1.StatsdIngestCapability.ShortName,
 		argName:   dynatracev1beta1.StatsdIngestCapability.ArgumentName,
 		Configuration: Configuration{
-			SetDnsEntryPoint:     true,
-			SetReadinessPort:     true,
-			SetCommunicationPort: true,
-			CreateService:        true,
+			SetDnsEntryPoint:       true,
+			SetReadinessPort:       true,
+			SetCommunicationPort:   true,
+			CreateService:          true,
+			CreateEecRuntimeConfig: true,
```

Comment:
why is this flag needed when it's only applicable for statsd? Do we need another check if we are using statsd?

Reply:
It's needed because the config map is only useful if there are relevant annotations. Most client deployments won't even see the `ConfigMap` for EEC or StatsD runtime properties.